api/base/serializers.py

```diff
@@ -328,6 +350,26 @@ def __init__(self, **kwargs):
         self.link_type = kwargs.pop('link_type', 'url')
         super(JSONAPIHyperlinkedGuidRelatedField, self).__init__(read_only=True, **kwargs)
 
+    def resolve(self, resource):
+        """
+        Resolves the view for target node or target comment when embedding.
+        """
+        embed_value = resource.target._id
+        if resource.target._name == 'node':
```

Comment:
Fair enough; we can always support multiple kwargs when the need arises.

The angle-bracket resolution should probably be made into a function rather than a method; it doesn't need to be coupled to`RelationshipField` (notice that there is no use of `self` in `lookup_attribute`.

I'm thinking

```python
resolve_attributes(node, {'node_id': '<_id>'})
```

But that's outside the scope of this PR.